lib/galaxy/datatypes/util/gff_util.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, reader, fields, chrom_col=0, feature_col=2, start_col=3, end_
         self.score = self.fields[self.score_col]
 
         # GFF attributes.
+        if 8 >= self.nfields:
```

Comment:
```suggestion
        if self.nfields < 9:
```

Reply:
I was using the same coding style of the code above, but we could probably update all conditions as you suggested.